frontend/integration-tests/views/networking.view.ts

```diff
@@ -0,0 +1,51 @@
+import { $, $$, browser, by, ExpectedConditions as until, element } from 'protractor';
+import * as crudView from '../views/crud.view';
+
+const name = $('#name');
+const selectServiceBtn = $('#service');
+const selectTargetPort = $('#target-port');
+const createRouteForm = $('.co-create-route');
+const saveButton = $('#save-changes');
+
+const chooseFromList = async(itemName: string) => {
```

Comment:
We might want to pull this into a file like `form.view.ts` so it can be reused elsewhere for any dropdown.

Reply:
Do we need a different file name for common views? I'm wondering if `getResourceJSON`, `isValueInJSONPath` in `secrets.view.ts` can be moved out as well